pkg/config/application.go

```diff
@@ -26,15 +26,15 @@ const (
 	defaultWaitApprovalTimeout  = Duration(6 * time.Hour)
 	defaultAnalysisQueryTimeout = Duration(30 * time.Second)
 	allEventsSymbol             = "*"
+	EnvLabelKey                 = "env"
```

Comment:
How about prefixing all reserved label keys with something like `pipecd.dev`?
So it should be `pipecd.dev/env`.

Reply:
I would say this is just a temporary workaround to encourage users to use labels.
In the future, the Environment concept will be removed, and this label will be just a label.
If you're looking into other reserved labels, it would be worth being prefixed.

Reply:
I see that the `env` key is just a temporary one until we can get rid of the Environment concept.
But I will vote for prefixing all revered labels by `pipecd.dev/`. There are some reasons behind my vote which I can say at this time:
- we don't have an exact timeline for removing the Environment concept yet
- prefixing will be good to avoid conflicting with the user-defined labels (doc to explain it could be easier, just say "all label keys prefixed by `pipecd.dev/` are reversed")
- it looks good as a convention for our future

Reply:
Makes sense. I'm happy to prefix it 👍

Reply:
Thank you. So let's do it. 💯

Reply:
Applied. PTAL!